### DISCUSSION TITLE
Release Candidate Two for Jamulus 3.10.0 is out now!

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#3153] [Announcements]
We're excited to announce the availability of the second release candidate for Jamulus 3.10.0. Translations are now finished and we think that we are soon in a releasable state.

Notes:
* Please note that Jamulus 3.10.0 will from now on only support Windows 10 and 11 due to the upgrade to Qt 6.
* The macOS legacy build is now considered deprecated and may be removed in future due to the lack of support for outdated Xcode versions on GitHub actions

Please find all the details and the downloads on the [GitHub releases page](https://github.com/jamulussoftware/jamulus/releases/tag/r3_10_0rc2).

Thanks to everyone who did their part to make this release happen:
- Code contributors (unfinished list): @ann0see, @comradekingu, @danryu, @declension, @hoffie, @ignotus666, @jujudusud, @mcfnord, @pljones, @Rob-NY
- Translators (unfinished list): @adoniasalmeida, @ann0see, @BLumia, @bugith, @C10udburst, @comradekingu, @cup113, @dzpex, @fnogcps, @gallegonovato, @gbonaspetti, @gnu-ewm, @ignotus666, @jerone, @mapi68, @MarongHappy, @melcon, @pljones, @SoulGI, @trebmuh, @Wdtwsm777, @yangyangdaji, @henkdegroot, @ignotus666, Iva Hristova
- ... and lots of people who brought new ideas or suggestions, guided their local colleagues or helped in various other ways!

Please test this release candidate. Feedback, questions and bug reports are appreciated!

**Comment by ann0see** [comment]
The macOS build is not notarized, so it will probably show an error. We are after it.

**Comment by dcorson-ticino-com** [comment]
Avast won't let me install 3.10rc2
![J3 10threat](https://github.com/jamulussoftware/jamulus/assets/64357360/589c6b59-11a2-49ea-8ce7-3891aed1862d)

**Reply by mcfnord** [reply]
[Here's a bit about that claim](https://www.alphr.com/what-is-idp-generic/).

**Reply by henkdegroot** [reply]
Seem you are one of the lucky ones....on Avast and AVG find something, all other major av software believes it is safe (checked using virustotal.com):

![image](https://github.com/jamulussoftware/jamulus/assets/13550012/41604aa5-e782-4beb-ac7b-7fe5013e7753)

**Reply by ann0see** [reply]
Not sure what to do against that

**Reply by dcorson-ticino-com** [reply]
Of course I can still install it, but how many users will jump through the hoops.
Unfortunately I think we need to find out where this is coming from.

**Reply by dcorson-ticino-com** [reply]
I sent Avast a false positive notice.  We will see if that helps.
After installing with an exception rc2 seems to be running fine.

**Reply by pljones** [reply]
I'd imagine many virus checkers will have a simple binary hash for the Jamulus.exe and any installer they've assured is clean.  Then any program "claiming" to be that exe or installer will come up as malware and won't even bother checking for injected code if that simple check fails.

All reporting it would then do (especially for the rc2, which will be rebuilt for release) is to postpone the issue that commonly happens at release for much software.

**Comment by rdica** [comment]
Servers updated and no issues experienced with a session tonight.

**Reply by pljones** [reply]
Non-"Any Genre" Directories now updated, too.

**Comment by ann0see** [comment]
I'm currently unsure if I can fix the macOS signing. We might need to get an alternative. Emlyn is currently unavailable.

**Comment by mcfnord** [comment]
I've heard people say they use an older version of Windows. Windows 8.1 lost Microsoft support on January 11th, so perhaps those computers shouldn't be on the internet, but I bet our userbase has a few die-hards. Is there a way to discourage them from upgrading to this release? I guess we'll cover this in our announcement. Will our installer stop and direct them if they try installing the newest bits?

**Reply by ann0see** [reply]
See https://github.com/jamulussoftware/jamulus/pull/2300 for a discussion on that. My opinion changed after the support drop of 8.1

**Reply by mcfnord** [reply]
A moderately skilled user running Windows 8 described her upgrade experience:

- She got the Windows build and installed it.
- It said there was a DLL missing.
- She found the previous version download in her Downloads folder, and re-installed it.

I asked her why she ran Windows 8. She said there's nothing on it except Jamulus, so it's not a security risk. (Not sure I agree, but it was interesting to hear.) Ideally, I wish we would warn her, and hope we still provide the previous build.

**Reply by ann0see** [reply]
the previous build is linked from the install page. On purpose it's not prominent.

![legacy builds paragraph](https://github.com/jamulussoftware/jamulus/assets/20726856/32959911-7c0b-4a5f-bc57-70f653b4b317)
